### ISSUE TITLE
Elixir not available to install though listed as supported.

### ISSUE THREAD

**Issue by georgealexanderday** [nvim-treesitter/nvim-treesitter#1335]
**Describe the bug**

Unable to install TS for elixir as it says it is a supported language in the readme. When trying to install it using :TSInstall it is not available as an option.

**To Reproduce**

1. Inside nvim type :TSInstall elixir
2. See error

**Expected behavior**

If it is supported, install elixir ts.

**Output of `:checkhealth nvim_treesitter`**

<details>
<code>

health#nvim_treesitter#check
========================================================================
## Installation
  - WARNING: `tree-sitter` executable not found (parser generator, only needed for :TSInstallFromGrammar, not required for :TSInstall)
  - OK: `node` found v14.17.0 (only needed for :TSInstallFromGrammar)
  - OK: `git` executable found.
  - OK: `cc` executable found. Selected from { vim.NIL, "cc", "gcc", "clang", "cl" }
  - OK: Neovim was compiled with tree-sitter runtime ABI version 13 (required >=13). Parsers must be compatible with runtime ABI.

## Parser/Features H L F I
  - toml           ✓ ✓ ✓ ✓ 
  - lua            ✓ ✓ ✓ ✓ 

</code>
</details>

**Output of `nvim --version`**

<details>
<code>

NVIM v0.5.0-dev+1345-g6dd04ed5f
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/gcc -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -Wp,-U_FORTIFY_SOURCE -Wp,-D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/builddir/build/BUILD/neovim-0.5.0~dev.1345.g6dd04ed5f/x86_64-redhat-linux-gnu/config -I/builddir/build/BUILD/neovim-0.5.0~dev.1345.g6dd04ed5f/src -I/usr/include -I/usr/include/lua-5.1 -I/builddir/build/BUILD/neovim-0.5.0~dev.1345.g6dd04ed5f/x86_64-redhat-linux-gnu/src/nvim/auto -I/builddir/build/BUILD/neovim-0.5.0~dev.1345.g6dd04ed5f/x86_64-redhat-linux-gnu/include
Compiled by mockbuild

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info

</code>
</details>

**Additional context**

Absolutely love the work you are doing and super excited that I may be able to use if for elixir!! :)

**Comment by stsewd** [comment]
Hi, what error do you get? I'm able to install elixir without problems. Maybe you are using `ensure_installed` and you don't have elixer listed?

**Comment by georgealexanderday** [comment]
It isnt listed as an option to be installed at all - only two languages listed beginning with e are elm and erlang using the tab completion. I am currently using ensure_installed "maintained" and when using :TSInstallInfo, elixir is also not listed.

Using :TSInstall elixir I get the error

<code> tree-sitter CLI is needed because `elixir` is marked that it needs to be generated from the grammar definitions to be compatible with nvim! </code>

I have installed tree-sitter-cli using npm but I guess it isnt installed correctly. Is there a specific location I should be installing it to?

**Comment by georgealexanderday** [comment]
Fixed by installing the latest version of tree-sitter-cli. The version listed on crates was out of date.

<code>cargo install tree-sitter-cli --git https://github.com/tree-sitter/tree-sitter.git --tag v0.19.5</code>

**Comment by aegatlin** [comment]
I think it is still worth suggesting that the discrepancy here between (1) `ensure_installed: 'maintained'`, (2) the fact that elixir is listed in the supported languages on the README as maintained, and (3) the fact that it's not present out-of-the-box, should be mentioned somewhere.

I'd be happy to submit some PR against the docs to help others who, like me, might have been confused in assuming it was included as a "maintained" member. I am currently beginning my journey into learning what tree-sitter-cli is and how to use it get elixir.

tl;dr: update the docs (and I would be happy to follow a maintainer's suggestion on how to go about doing so myself in a PR).
